src/Symfony/Component/Validator/ConstraintViolationInterface.php

```diff
@@ -36,7 +36,7 @@ interface ConstraintViolationInterface
     /**
      * Returns the violation message.
      *
-     * @return string The violation message
+     * @return string The violation message as a string or a stringable object
```

Comment:
let's do the same on https://github.com/symfony/symfony/pull/31083#issuecomment-516569294 if you can confirm it's required with your issue to be solved?

Reply:
Done.